### DISCUSSION TITLE
please Tell me your favorite game

### DISCUSSION THREAD

**Discussion by zhangyongxin1212** [NAIST-SE/AttractingContributors#134] [General]
https://github.com/zhangyongxin1212/AttractingContributors-Game

**Comment by GuangxianZhu** [comment]
I like CSGO.

**Comment by MatsumuraTaiki** [comment]
Added, please help me too
https://github.com/MatsumuraTaiki/AttractingContributors-Actors

**Comment by ellielo** [comment]
"Ice scream 4" from [gamesdizi](https://gamesdizi.com/) is awesome. Mistery and horror combined with the cool graphics and interesting tasks. 10 of 10 for me. "Resident evil village", GTA6, Run of life - those are also good. I never liked horror movies (watched though with my BF from time to time) but turned out i love horror games. Very surprised of myself)

**Comment by ElefunAi** [comment]
I have added Game list!
Please check and help me!

https://github.com/ElefunAi/AttractingContributors-texteditor

**Comment by YPanY** [comment]
added Mass Effect, pls help me by adding your favorite programming language or framework here👇 THX!
https://github.com/Pan-NLP-NAIST/AttractingContributors-programming_language/blob/main/README.md

**Comment by t-matsumoto4758** [comment]
Added!
please help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Comment by Gavin-23** [comment]
Added!
Please help me too~~
Thanks 😊
https://github.com/Gavin-23/AttractingContributors-Music
